persistence/rocksdb/src/main/java/org/infinispan/persistence/rocksdb/RocksDBStore.java

```diff
@@ -358,6 +368,34 @@ public MarshalledEntry load(Object key) {
         }
     }
 
+    @Override
+    public void writeBatch(List<MarshalledEntry<? extends K, ? extends V>> marshalledEntries) {
+        try {
+            WriteBatch writeBatch = new WriteBatch();
```

Comment:
I've created [ISPN-7912](https://issues.jboss.org/browse/ISPN-7912) to track this, I'll issue a fix in a separate PR.